AttitudeManager/Inc/AM.hpp

```diff
@@ -34,7 +34,9 @@ class AttitudeManager {
     uint8_t desired_controller_index = 0;
     const ControlInterfaceList controller_interfaces;
     static constexpr uint64_t transition_time_ms = 5000;
+    static constexpr uint64_t desired_airspeed = 60; 
```

Comment:
Why is this constexpr? wouldn't it change depending on the control algorithm that you are transitioning to?